_sources/Unit5-Writing-Classes/topic-5-1-parts-of-class.rst

```diff
@@ -37,49 +37,131 @@
 Anatomy of a Java Class
 =======================
 
-In Unit 2, we learned to use **classes** and **objects** that are built-in to Java or written by other programmers. In this unit, you will learn to write your own classes and objects!
+In Unit 2, we learned to use **classes** and **objects** that are built-in to
+Java or written by other programmers. In this unit, you will learn to write your
+own classes and make your own objects!
 
-Creating a Class
-------------------
+Remember that a **class** in Java defines a blueprint for creating objects. When
+you create **objects**, you create new **instances** of that class and what you
+can do with those instances is determined by what methods are defined in the
+class. Also recall that objects are always accessed by **references**—the values
+we actually store in variables or pass when we call methods are references to
+the objects so we can have different variables that refer to the same object.
```

Comment:
Reword sentence? Maybe split up or leave off "so" clause for now.

Reply:
Actually I just took out the sentence altogether. I don't think there's really any reason to talk about this here.